src/components/common/Layout/components/Account/components/AccountModal/components/LinkMoreSocialAccount.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Avatar,
+  Button,
+  Collapse,
+  HStack,
+  Icon,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react"
+import useConnectPlatform from "components/[guild]/JoinModal/hooks/useConnectPlatform"
+import platforms from "platforms/platforms"
+import { useEffect } from "react"
+import { PlatformName } from "types"
+import capitalize from "utils/capitalize"
+
+type Props = {
+  platformName: string
+}
+
+const LinkMoreSocialAccount = ({ platformName }: Props): JSX.Element => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  const { onConnect, isLoading, response } = useConnectPlatform(
+    platformName as PlatformName,
+    onClose
+  )
+  useEffect(() => onOpen(), [platformName])
```

Comment:
we should add {defaultIsOpen: true} to useDisclosure instead of this. If you've tried that and it didn't work, we should add a comment for a hacky useEffect like this

Reply:
{defaultIsOpen: true} doesn't works, when i disconnect an account, gonna add a comment